modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-increase-max-vols-per-node-overview_{context}"]
+= Increasing maximum volumes per node for vSphere
+
+For VMware vSphere version 7, {product-title} restricts the maximum number of volumes per node to 59. 
+
+However, for vSphere version 8 or later, you can increase the allowable number of volumes per node to a maximum of 255. Otherwise, the default value remains at 59.
+
+.Limitations
+* You must be running VMware vSphere version 8 or later.
+
```

Comment:
There's also another potential pitfall that users should be aware of which is the overall limit per node that they can hit when using this feature: https://github.com/openshift/enhancements/pull/1748/files#diff-a45d3a5e19d424ae3457a5db5722c6b7511bf3662941972a2d8db77edf272f7dR118-R121

Reply:
Done.